action.yml

```diff
@@ -10,7 +10,7 @@ inputs:
   comment:
     description: The comment to place in the PR
     default: |
-      ⚠️ This PR contains unsigned commits. To get your PR merged, please sign those commits (`git commit -S --amend --no-edit`) and force push them to this branch (`git push --force-with-lease`).
+      ⚠️ This PR contains unsigned commits. To get your PR merged, please sign those commits (`git rebase --exec 'git commit -S --amend --no-edit -n' @{upstream}`) and force push them to this branch (`git push --force-with-lease`).
```

Comment:
`[non-blocking, learning]`

I did read the documentation for [`git rebase`](https://git-scm.com/docs/git-rebase), [`git commit`](https://git-scm.com/docs/git-commit) and [gitversions](https://git-scm.com/docs/gitrevisions) to better understand the combined command you provide. 

Could you confirm if the following steps reflect what the command does?
1. `--exec` flag passes to `git rebase` the command that will be executed for the process
2. We pass the `git commit` command so that we commit each one without interaction
3. `--no-edit` is used to keep the previous commit message.
4. `-n` is to disable pre-comit hooks. Since we just want to sign the commits, they can be skipped since they were in place when the original commits were made.
5. `--amend` can be used in combination with `git rebase` and it's for "adjusting" the commits rather than fully remake them (in this case we just sign them).
6. `@{upstream}` is used to ensure all commits are part of the rebase process.

Reply:
Yes! 🙌